dev/05_data_core.ipynb

```diff
@@ -669,7 +669,7 @@
     "               for i,(b,s,dk) in enumerate(zip(bss,shuffles,dl_kwargs))]\n",
```

Comment:
<p>Please keep the names private for classes and functions only used in tests, as we don't want to pollute the state too much :)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/fastai/fastai_dev/pull/293/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='52'/>